docs/ref/contrib/contenttypes.txt

```diff
@@ -416,6 +417,12 @@ relationships::
     >>> b.tags.set([t1, t3])
     >>> b.tags.all()
     <QuerySet [<TaggedItem: django>, <TaggedItem: Web development>]>
+    >>> b.tags.get_or_create(tag="django")
+    <QuerySet [<TaggedItem: django>, <TaggedItem: Web development>]>
+    >>> b.tags.update_or_create(tag="dynamic")
+    <QuerySet [<TaggedItem: django>, <TaggedItem: Web development>, <TaggedItem: dynamic>]>
+
+Async versions of the above are supported with an ``a`` prefix.
```

Comment:
Related to my other unresolved comment, I don't really know what I'm doing when it comes to the Django docs so I'm happy to have someone wordsmith this for me

Reply:
IMO this changes are unnecessary, reverted.